### ISSUE TITLE
Improve Logo

### ISSUE THREAD

**Issue by theartist007** [thealphadollar/messiah#38]
Although the current logo looks decent (all hail @themousepotato ❤️ ), we would want a colourful, graphic version of the logo. 
Use your imagination, design skills, and show us what you have got..!!

**Comment by Radhika007** [comment]
How about this logo? 
![messiah logo](https://user-images.githubusercontent.com/37405390/49374608-e2767780-f727-11e8-8f85-9ebe7ed36da9.png)

**Comment by theartist007** [comment]
Decent. 
- I would want to change the light blue color though. It's looking kinda bleak. Change it to something colorful.
- Try to develop a png version of the image too, without the white background, coz we are going to put this up on the main website too.
Feel free to ping me.

**Comment by Radhika007** [comment]
Okay, so here are some options for the logo. (No white background is there)
![logo1](https://user-images.githubusercontent.com/37405390/49489217-01802100-f870-11e8-9b68-574ca04c0762.png)
![logo2](https://user-images.githubusercontent.com/37405390/49489225-06dd6b80-f870-11e8-98d7-e61fd5c0f9ea.png)
![logo3](https://user-images.githubusercontent.com/37405390/49489235-0f35a680-f870-11e8-8ca1-7596eb1c65a1.png)

**Comment by thealphadollar** [comment]
The last one looks good to me; send a PR.

@theartist007 I don't think we should close the issue since there is always room for improvement and we will merge a better logo if someone comes up with it.

**Comment by Radhika007** [comment]
There are few more options, if you want to take a look?
I hope it's alright if we put up an abstract shape instead of an icon?

**Comment by theartist007** [comment]
@Radhika007 , yes, you can show the other options. The last one looks decent though. Umm, I think you do realise that the main website has a maroon background, so the colours should be chosen, so that it matches quite well, with the background. 
Also, we need another logo image (just add black background to the one chosen above). This will be updated on the README.md file.

@thealphadollar  Yes, we won't close this.

**Comment by satyamroy001** [comment]
IAM in kwoc program.. I want to develop a logo.

**Comment by theartist007** [comment]
@satyamroy001, Radhika has already contributed a new logo. So, it won't be changed for now. You can approach us with a new logo, maybe in the next program, or KWOC. 
Thanks for your interest :)

**Comment by satyamroy001** [comment]
If not tell me what to do ....to improve Messiah.(I don't know CSS,html etc) only c programming

**Comment by theartist007** [comment]
@satyamroy001 , I would recommend you to join our slack channel and discuss further options. Github Issue comments isn't exactly a place to do that :)

**Comment by satyamroy001** [comment]
Can you plz post the link of slack channel???????

**Comment by swghosh** [comment]
Join the chat in kwoc-messiah/Lobby
https://gitter.im/kwoc-messiah/Lobby?utm_source=ios&utm_medium=link&utm_campaign=ios-share-link
